examples/reference/widgets/Button.md

```diff
@@ -0,0 +1,116 @@
+```python
+import panel as pn
+import panel_material_ui as pmu
+
+pn.extension()
+```
+
+## Button Intro
+
+Buttons allow users to take actions, and make choices, with a single tap.
+
+Buttons communicate actions that users can take. They are typically placed throughout your UI, in places like
+
+- Modal windows
+- Forms
+- Cards
+- Toolbars
+
+Then `panel-material-ui` button is built on top of the [Material UI `Button`](https://mui.com/material-ui/react-button/).
+
+```python
+import panel as pn
+import panel_material_ui as pmu
+from panel_material_ui import Button
+
+pn.extension()
+```
+
+## Basic Button
+
+The Button comes with three *variants*: text (default), contained, and outlined.
+
+```python
+pn.Row(
+    Button(label="Text", variant="text"),
+    Button(label="Contained", variant="contained"),
+    Button(label="Outlined", variant="outlined"),
+).servable()
+```
+
+## Text Button
+
+[Text buttons](https://m2.material.io/components/buttons#text-button) are typically used for less-pronounced actions, including those located: in dialogs, in cards. In cards, text buttons help maintain an emphasis on card content.
+
+```python
+pn.Row(
+    Button(label="Primary"),
+    Button(label="Disabled", disabled=True),
+    Button(label="Link", href="#text-buttons"),
+).servable()
+```
+
+## Contained button
+
+[Contained buttons](https://m2.material.io/components/buttons#contained-button) are high-emphasis, distinguished by their use of elevation and fill. They contain actions that are primary to your app.
+
+```python
+pn.Row(
+    Button(label="Contained", variant="contained"),
+    Button(label="Disabled", variant="contained", disabled=True),
+    Button(label="Link", variant="contained", href="#contained-buttons"),
+).servable()
+```
+
+You can remove the elevation with the `disable_elevation` parameter.
+
+```python
+Button(label="Disable elevation", variant="contained", disable_elevation=True).servable()
+```
+
+## Outlined Button
+
+[Outlined buttons](https://m2.material.io/components/buttons#outlined-button) are medium-emphasis buttons. They contain actions that are important but aren't the primary action in an app.
+
+Outlined buttons are also a lower emphasis alternative to contained buttons, or a higher emphasis alternative to text buttons.
+
+```python
+pn.Row(
+    Button(label="Primary", variant="outlined"),
+    Button(label="Disabled", variant="outlined", disabled=True),
+    Button(label="Link", variant="outlined", href="#outlined-buttons"),
+).servable()
+```
+
+**MORE TO COME. NOT FINISHED**
+
+## Aliases
+
+For backwards compatibility with Panel certain parameters are allowed as aliases:
+
+- **`button_style`**: Alias for `variant`
+- **`button_type`**: Alias for `color`
+- **`name`**: Alias for `label`
+
+```python
+Button(name="Alias", button_style="outlined", button_type="success").servable()
+```
+
+## API
+
+### Parameters
+
+```python
```

Comment:
I think this is much more valuable and easy to maintain than the written `API` docs we normally have.

If we could write out the parameter `doc` values instead of having them as tooltips I think it might be valuable.

Reply:
You replied in Discord @philippjfr 

![image](https://github.com/user-attachments/assets/dbb4ba7e-30f9-4517-b5fb-3d582ac9b5f5)

I don't know if you understand that the API parameters are still there. Just all of them and as a table.

https://github.com/user-attachments/assets/dd431960-0a37-40af-b8e5-84b840b98471

I think its import to move API docs down or some separate place. The current reference guides do not sell Panel/ Panel Material UI well. The first impression is very text heavy. Maybe later the API docs is valuable. But you risk loosing potential users because they don't see anything nice when quickly exploring the components provided.

Reply:
Should I add back the missing API parameter documentation in the beginning. And just have the control in the end? Then we can take discussion about this another time and just focus on getting Button fixes and docs improments in for now?

Reply:
That sounds great to me. We do need to tackle the API docs problem but not immediately.